lms/djangoapps/commerce/utils.py

```diff
@@ -55,6 +55,14 @@ def get_absolute_ecommerce_url(self, ecommerce_page_url):
         """
         return urljoin(self.ecommerce_url_root, ecommerce_page_url)
 
+    def get_order_dashboard_url(self):
+        """ Return the URL to the ecommerce dashboard orders page.
+
+        Returns:
+            order dashboard url.
+        """
+        return self.ecommerce_url_root + '/dashboard/orders/'  # TODO: Store this in settings?
```

Comment:
@reviewers Could use advice on whether this belongs in this file or if dashboard/orders should be somewhere in a settings file

Reply:
I'd say to put it in a settings file

Reply:
Yeah, if it's in this file maybe it should be a variable instead?

Reply:
I'm leaning towards a setting. I haven't added a setting before though, seems like theres something about a model for the ecommerce configuration.